vernac/comAssumption.ml

```diff
@@ -153,7 +153,7 @@ let do_assumptions kind nl l =
     let env =
       push_named_context (List.map (fun (_,id) -> LocalAssum (id,t)) idl) env in
     let ienv = List.fold_right (fun (_,id) ienv ->
-      let impls = compute_internalization_data env Variable t imps in
+      let impls = compute_internalization_data env sigma Variable EConstr.(of_constr t) imps in
```

Comment:
Should we make `interp_assumption` return an econstr instead?

Reply:
Good catch.